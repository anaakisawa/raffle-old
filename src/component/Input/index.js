import React, { Component } from 'react';

class Input extends Component {
  render () {
    return (
	  <div className="Input">
	    P<input type = "text">
	  </div>

			);
  }

}

export default Input;
