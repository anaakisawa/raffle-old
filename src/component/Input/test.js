import React from 'react';
import Input from './index';
import renderer from 'react-test-renderer';

describe('Component - App', () => {
  test('renders an input element', () => {
    const component = renderer.create(
      <Input/>,
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
});
