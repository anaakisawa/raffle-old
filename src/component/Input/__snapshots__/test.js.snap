// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - App renders an input element 1`] = `
<div
  className="Input"
>
  <input
    type="text"
  />
</div>
`;
